test/spec/utils/alt-resolver.test.js

```diff
@@ -36,7 +37,8 @@ describe('Alt Resolver', () => {
 
   it('should render async a dummy component', (done) => {
     (async function () {
-      const content = await altResolver.render(Dummy, flux, true);
+      const Stubbed = stubApp(flux)(Dummy);
+      const content = await altResolver.render(Stubbed, flux, true);
```

Comment:
'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).<br>Missing semicolon.